scipy/optimize/_linprog_ip.py

```diff
@@ -837,16 +840,16 @@ def eta(g=gamma):
 
 def _linprog_ip(
         c,
-        c0=0,
-        A=None,
-        b=None,
-        callback=None,
-        _T_o=[],
-        alpha0=.99995,
-        beta=0.1,
+        c0,
+        A,
+        b,
```

Comment:
c, A, and b are part of the problem definition, so priority seems higher than callback and options

Reply:
This is more of a style issue, is it worth having every argument on its own line?

Reply:
No. But it's been that way since autopep8 made it like that in the original version, I think, so I didn't think to change it. Fixed in `_linprog_ip` and `_get_delta`.